src/features/import-maps.js

```diff
@@ -50,10 +53,12 @@ function processScripts () {
 }
 
 function extendImportMap (importMap, newMapText, newMapUrl) {
+  var newMap = {};
   try {
-    var newMap = JSON.parse(newMapText);
+    newMap = JSON.parse(newMapText);
   } catch (err) {
-    throw Error(process.env.SYSTEM_PRODUCTION ? errMsg(1) : errMsg(1, "systemjs-importmap contains invalid JSON"));
+    console.error(Error(process.env.SYSTEM_PRODUCTION ? errMsg(1) : errMsg(1, "systemjs-importmap contains invalid JSON")));
```

Comment:
Might make sense to move SystemJS error 1 to be a warning instead of an error in the documentation. But doing so would be a breaking change, so not worth it now.

Reply:
Changing it to a warning may well be the right approach. Is it really a breaking change to do that? This PR is already effectively alters the behaviour from an error to a warning anyway.

Reply:
Code using older versions of systemjs would continue linking to the old URL

Reply:
We versioned the doc URL shortlinks though right? So we could then bump the docs link version.

Reply:
No, there's no version in the doc URL shortlinks.

Reply:
Ok, but at least the numbering scheme means the links are permalinks so that as long as the numbers stay the same users will get the right information. What about that would be breaking?

Reply:
Current URL: https://git.io/JvFET#1

New URL if we make it a warning: https://git.io/JvFET#W4

We can avoid it being a breaking change if we duplicate the documentation in both places. But removing Error 1 since it's now W4 would be breaking.

Reply:
Ah I finally see what you mean, thanks.

Ok, then just a duplication or a redirecting pointer link would be fine I think.

Reply:
I've copied error 1 to become W5 and changed the logging code accordingly. I've also switch from logging the broken JSON as a string to a simple object. This makes the output more readable of the JSON is large because browser devtools collapse objects if too large.